Pipeline/SimpleBuildReportDeploy/README.md

```diff
@@ -0,0 +1,293 @@
+# Simple Build Report Deploy (Deploy the package to target libraries)
+## Description
+This "SimpleBuildReportDeploy.groovy" sample script is capable of untaring the tar package produced by [PackageBuildOutputs](../PackageBuildOutputs/) and deploy the tar package contents to the target libraries. 
+
+Potential scenarios include:
+* Deploying the tar package to the target environment libraries.
+* If the user wants to do shift-left testing using IBM Z Virtual Test Platform, and subsystem updates are not required.
+
+As of now, this script does not perform any activation activities like a CICS NEWCOPY or a DB2 BIND.  Currently this script does not support rollback process also.
+
+### Prerequisites
```

Comment:
We need to state, that it only works for packages of PackageBuildOutputs based on a single BuildReport.json and does not support cumulative packages based on multiple build reports.

Reply:
The code deploys for multiple build reports as well.  The code used is 
```
// Find the BuildReport.json file for deployment 
listDirFiles.eachFile {
   if(it.getName().contains("BuildReport.json")) {
	   if(!it.exists()){
		    println("** Build report data at ${it} not found")
		    System.exit(1)
	   }
	   buildReportFlag = 1
	   deployFromBuildReport(it,tarExtractDirName,targetLibLLQMap,copyModeMap)
   }
}
```

The same is mentioned in the README file as well.  Please let me know if you are referring to something else.

Reply:
Please ignore #148 and consider #149 .